modules/mobile-agents/android/client/src/org/wso2/mdm/agent/api/Root.java

```diff
@@ -73,9 +74,9 @@ public boolean checkRootByBuildTags() {
 	public boolean checkRootBySuperUserApk() {
 		File suApk = new File(SU_APK);
 		if (suApk != null && suApk.exists()) {
-            		if (Constants.DEBUG_MODE_ENABLED){
-                		Log.d(TAG, "Super apk found");
-            		}
+			if (Constants.DEBUG_MODE_ENABLED){
```

Comment:
Fixed in Commit [https://github.com/wso2/product-mdm/commit/b0c66506d24fb5288300bec4224f1eaa58c7b426]